modules/ocf_mail/files/spam/logging/examine-mail-log

```diff
@@ -50,28 +50,12 @@ def extract_user(addr):
         return match.group(1)
 
 
-def is_vhost_host(host):
-    """Returns whether host (either IP or FQDN with trailing dot) hosts
-    vhosts."""
-
-    return host.lower() in ('death.ocf.berkeley.edu.', '169.229.226.23')
-
-
-def is_vhost_domain(domain):
-    """Returns whether domain is one of our vhosts."""
-
-    try:
-        answer = dns.resolver.query(domain, 'A')
-        return any(is_vhost_host(str(record)) for record in answer)
-    except (dns.resolver.NoAnswer, dns.resolver.NXDOMAIN):
-        return False
-
-
 def examine_mail(mail):
     """Examine a JSON dict of mail, yielding problem tuples."""
 
     from_name, from_addr = mail['from']
     from_domain = extract_domain(from_addr)
+    sender = mail.get('sender')
```

Comment:
someone is going to look at this a year from now and have no idea what the difference between "from" and "sender" are.

can we call this "authenticated_sender" or "smtp_user" or something less ambiguous?